modules/eventing/pages/eventing-handler-multiCollectionEventing.adoc

```diff
@@ -0,0 +1,160 @@
+= Function: Multi Collection Eventing
+:description: pass:q[Show how to access the Data Service when Eventing is listening to multiple collections.]
+:page-edition: Enterprise Edition
+:tabs:
```

Comment:
No real need for this attribute.

Reply:
It's just how my other scriptlets were done, I probably use it as it bolds things.